Modules/Scripted/VPAWModel/VPAWModel.py

```diff
@@ -312,6 +314,24 @@ def updateParameterNodeFromGUI(self, caller=None, event=None):
 
         self._parameterNode.EndModify(wasModified)
 
+    def onHomeButton(self):
+        """
+        Switch to the "Home" module when the user clicks the button.
+        """
+        with slicer.util.tryWithErrorDisplay(
+            "Failed to compute results.", waitCursor=True
+        ):
+            slicer.util.selectModule("Home")
+
+    def onVPAWVisualizeButton(self):
+        """
+        Switch to the "VPAW Visualize" module when the user clicks the button.
+        """
+        with slicer.util.tryWithErrorDisplay(
+            "Failed to compute results.", waitCursor=True
```

Comment:
same with this error message and the one just above it

Reply:
Both fixed